doc/build/dts/bindings-upstream.rst

```diff
@@ -208,3 +208,28 @@ Some examples of what **not** to do, and why:
        # This is the recommended value chosen by the manufacturer.
        default: 2
        description: baz time in milliseconds
+
+The ``zephyr,`` prefix
+**********************
+
+You must add this prefix to property names in the following cases:
+
+- Zephyr-specific extensions to bindings we share with upstream Linux. One
+  example is the ``zephyr,vref-mv`` ADC channel property which is common to ADC
+  controllers defined in :zephyr_file:`dts/bindings/adc/adc-controller.yaml`.
+  This channel binding is partially shared with an analogous Linux binding, and
+  Zephyr-specific extensions are marked as such with the prefix.
+
+- Configuration values that are specific to a Zephyr device driver. One example
+  is the ``zephyr,lazy-load`` property in the :dtcompatible:`ti,bq274xx`
+  binding. Though devicetree in general is a hardware description and
+  configuration language, it is Zephyr's only mechanism for configuring driver
+  behavior for individual devices. Therefore, as a compromise, we do allow some
```

Comment:
Perhaps replace "individual devices" with "individual device driver instances"?

Reply:
Personally, I have never been a fan of calling each `struct device` a "device driver instance". IMO this is a longstanding wart in our device driver model documentation. It's a `struct device`, so I try to be consistent about calling them "devices".

Reply:
My problem with using just "devices" is that it is ambiguous - at least to me. Not a show-stopper.

Reply:
OK, I think this should suit us both:

```
behavior for an individual ``struct device``
```